### ISSUE TITLE
Using F1 Score and classification_report on Multilabel

### ISSUE THREAD

**Issue by diogocortiz** [ThilinaRajapakse/simpletransformers#1033]
I tried to use F1-score in Multilabel model, but it didn't work. I received a message stating F1 could not be used.

Any cues on this?

**Comment by roshan-k-patel** [comment]
Mine is multiclass but this should still be applicable:

`model.train_model(df_train, eval_df=df_eval, accuracy=sklearn.metrics.accuracy_score, f1_macro = lambda truth, predictions: sklearn.metrics.f1_score(
    truth, predictions, average='macro'), f1_weighted = lambda truth, predictions: sklearn.metrics.f1_score(
    truth, predictions, average='weighted'))`
